account_security_extra/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_account_user,account.account user,account.model_account_account,base.group_user,1,1,1,0
```

Comment:
Group too permissive, and not group mentioned in module `DESCRIPTION`, better this one (read permission is actually available, see  https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/account/security/ir.model.access.csv#L11, but it could be kept here anyway)
```suggestion
access_account_account_user,account.account user,account.model_account_account,account.group_account_user,1,1,1,0
```